interface/src/app/analyze/search/searchDetail.js

```diff
@@ -18,14 +16,19 @@ angular.module('adage.analyze.detail', [
 
       show: function(search_item) {
         $scope.detail.search_item = search_item;
-        $scope.detail.showing = true;
-        $timeout(function() {
-          // if we inititate the scroll too early, the search results list
-          // hasn't finished compressing width and it doesn't work right,
-          // so this timeout waits until the transition has finished
-          // before scrolling
-          $scope.analyze.scroll_to_id(search_item.pk);
-        }, 500);
+        if (search_item.item_type === 'experiment') {
```

Comment:
So the idea is if the matched item of the search is an experiment, the experiment page will be loaded? Is this a new logic? I don't see it in original code.

Reply:
This should be more clear from the altered workflow on the test server: http://greenelab.fgtech.com/ (also shown in the workflow design sketch at https://github.com/greenelab/adage-server/issues/268).

Previously, when a user clicked on a search result, the experiment details would slide in from the right to appear within the `analyze` view. This made it complicated to parameterize the state of that page and my observations of users on the system showed the split-screen view to be of dubious value anyway.

Now, when a user clicks on an experiment in the search results, the `ui-router` is engaged to switch state so the experiment details load in their own page. (For now, clicking a sample in the search results retains the original behavior, but this will also change when that component is refactored.)